src/remote_gym/remote_environment.py

```diff
@@ -54,6 +83,7 @@ def __init__(
         self,
         url: Text,
         port: int,
+        remote_args: RemoteArgs = {},
```

Comment:
1. Can we rename this to `additional_remote_args`?
2. Can we create an example for passing `additional_remote_args` in "start_environment_interaction.py"?

Reply:
I believe additional is redundant and slightly misleading (its not additional, its overriding technically). Will add an example.